lib/ex_rose_tree.ex

```diff
@@ -199,12 +199,12 @@ defmodule ExRoseTree do
   """
   @doc section: :children
   @spec has_child?(t(), (t() -> boolean())) :: boolean()
-  def has_child?(%__MODULE__{children: children}, predicate) when is_function(predicate) do
+  def has_child?(%__MODULE__{children: children} = _tree, predicate) when is_function(predicate) do
```

Comment:
arguable as to value of `= _tree` here since it's fairly well-implied by `%__MODULE__{}`

Reply:
Can leave though, it doesn't hurt

Reply:
Yeah from the docs perspective it was already clear what type the arg is for this, the changes like this were just for consistency in terms of how the docs read, as with the changing of the args in the `Zipper` functions from `z` to all be `zipper`